google-cloud-bigtable/lib/bigtable.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "logger"
+
+module Bigtable
```

Comment:
All of the `google-cloud-*` libraries use the `Google::Cloud` namespace. I believe this should be `Google::Cloud::Bigtable`.

Reply:
@blowmage got it. I will update namespace.